### PR TITLE
[TS] LPS-159458 Restore Read more link with Liferay Learn article

### PR DESCRIPTION
**Pull request by ericyanLr**
Hi Team,

Previous PRs: 
- https://github.com/liferay-lima/liferay-portal/pull/3320
- https://github.com/liferay-lima/liferay-portal/pull/3325

/cc @AliciaGarciaGarcia 

**Update**
When referencing Liferay Learn articles, we actually have a module to handle those links.

Comments from Brian on the previous PR: https://github.com/brianchandotcom/liferay-portal/pull/121239#issuecomment-1204561518

Brian helped us declare the webdav link for our Liferay Learn article under this commit:
https://github.com/brianchandotcom/liferay-portal/commit/7a5f80b41514efed36f13bed4557cfea3f19f666
- He also mentioned: Feel free to change the message, see liferay-portal/learn-resources/*.json

**Proposed Changes**
This PR utilizes the Liferay Learn link by adding it directly to the **Access from Desktop** jsp page, instead of trying to handle the addition within a language key.

From what I can tell, the 'Read more' link is only being utilized for this page. The only concern may be how translations would be affected by using this approach.
- If translations is a concern, we can use our previous approach with the language keys and just utilize the LearnMessage to inject the proper URL.

Please let me know if you have any questions or would like me to make any additional changes. 

Thanks!

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp [hunk 1]
```diff
@@ -30,6 +30,11 @@ DLAccessFromDesktopDisplayContext dlAccessFromDesktopDisplayContext = new DLAcce
 	<div class="portlet-document-library">
 		<liferay-ui:message key="<%= dlAccessFromDesktopDisplayContext.getWebDAVHelpMessage() %>" />
 
+		<liferay-learn:message
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/access_from_desktop.jsp:33]: @ericyanLr 
I did not know about this component.
Interesting!

Thanks for the info :)

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
the ci seem stuck, I will test to close and open this PR and see if the ci reacts

If not, please @ericyanLr resend this PR with another name branch (no more  changes needed)

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**